public/styles/site/leaderboards.sass

```diff
@@ -25,22 +26,33 @@ input
   padding: 0
   ul
     list-style: none
-    direction: rtl
-
     li
       padding: 0.8em 0
-      // This is in here for the js
-      transform: rotate(180deg)
 
+.leaderboardFilter
+  border: 1px solid white
+  font-family: variables.$family-paragraph
+  font-size: var(--paragraph-font-size)
+  color: variables.$color-active
+  background-color: variables.$background-tertiary
+  padding: 0.5em 0 0.5em 0.5em
+  margin: 0 0.8em 
+  text-align: center
+  option
+    color: variables.$color-active
+    background-color: variables.$background-secondary
+      
+// This is in here for the js since the leaderboard is actually starting from 100 to 1. So it needs to be displayed in reverse.
+.rotateMe180
+  ul
+    li
+      transform: rotate(180deg)
```

Comment:
ouch! Is that the best thing we can do? Transform rotate is quite expensive
You can reverse order elements with a flexbox for instance, but there are other ways to do it in CSS
but do we even need to do it in CSS? Can't we just reverse the JS array before displaying...?